lib/pages/leaderboard_page/components/top_container.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:flutter/material.dart';
+import 'package:dalal_street_client/theme/colors.dart';
+
+Container topContainer(int myRank, List<Map<String, String>> tabledataOverall) {
+  return Container(
+    padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 10),
+    decoration: BoxDecoration(
+      color: background2,
+      borderRadius: BorderRadius.circular(10),
+    ),
+    width: double.infinity,
+    child: Center(
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        children: [
+          Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Stack(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(top: 35.0),
+                    child: Container(
+                        height: 80,
+                        width: 80,
+                        decoration: BoxDecoration(
+                            shape: BoxShape.circle,
+                            border: Border.all(color: Colors.white, width: 3),
+                            image: const DecorationImage(
+                                image: AssetImage(
+                                    '../../../assets/images/placeholder.png'),
```

Comment:
```suggestion
                                    'assets/images/placeholder.png'),
```
should be like this ig, none of the assets are loaded with this file format

Reply:
do check with other places also